modules/logging-release-notes-5-7-14.adoc

```diff
@@ -0,0 +1,24 @@
+// module included in logging/logging-5-7-release-notes
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-7-14_{context}"]
+= Logging 5.7.14
+This release includes link:https://access.redhat.com/errata/RHSA-2024:2930[OpenShift Logging Bug Fix 5.7.14].
+
+
+[id="logging-release-notes-5-7-14-bug-fixes"]
+== Bug fixes
+
+* Before this update, an issue in the metrics collection code of Logging Operator caused it to report stale telemetry metrics. With this update, a rewrite of the metrics code resolves the issue. (link:https://issues.redhat.com/browse/LOG-5472[LOG-5472])
```

Comment:
Same comment as other PR with this text - you don't have to say "a rewrite of the metrics code", just say what the Logging Operator does now after the update. But not requesting, just recommending. Your call.

Reply:
I understand your point. I will simply change the last sentence to: With this update, the Logging Operator does not report stale telemetry metrics.